src/components/Footer/index.css

```diff
@@ -0,0 +1,20 @@
+@import "../../styles/colors.css";
+@import "../../styles/typography.css";
+@import "../../styles/layout.css";
+
+.footer {
+    padding: 25px 0;
+    text-align: center;
+    color: rgba(255, 255, 255, 0.7);
+    background-color: $color-fire-red;
+    position: fixed;
+    width: 100%;
+    left: 0;
+    bottom: 0;
+}
+
+.tagline {
+    font-size: 14px;
+    letter-spacing: 1px;
+    margin: 0;
+}
```

Comment:
You're missing a newline here

Reply:
True. What do you think of adding a editorconfig with that specification so we never forget that again?

Reply:
We don't need that, as we already have [stylelint] (https://github.com/stylelint/stylelint). Just install linter in your editor and it will automatically format it for you on save. You can install also prettier.

Reply:
I think if we had a editorconfig it could specify tab size and al sort of editor properties.

Reply:
We don't need that. Eslint, stylint and prettier already take care of all of that. I'll explain you in person in the next meeting if you need.